sdk/feature-management/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/feature-management",
-  "version": "1.0.0-preview",
+  "version": "1.0.0-preview.1",
```

Comment:
Is there any name convention for the version? Or we just followed the appinsights?

Reply:
Convention:
JS FM: https://www.npmjs.com/package/@microsoft/feature-management/v/1.0.0-preview.1

JS provider: https://www.npmjs.com/package/@azure/app-configuration-provider?activeTab=versions